allennlp/training/trainer.py

```diff
@@ -46,13 +47,11 @@ def sparse_clip_norm(parameters, max_norm, norm_type=2):
     concatenated into a single vector. Gradients are modified in-place.
     Supports sparse gradients.
 
-    Arguments
-    ---------
-        parameters (Iterable[Variable]): an iterable of Variables that will have
-            gradients normalized
-        max_norm (float or int): max norm of the gradients
-        norm_type (float or int): type of the used p-norm. Can be ``'inf'`` for
-            infinity norm.
+    Parameters
+    ----------
+    parameters : ``(Iterable[Variable])`` an iterable of Variables that will have gradients normalized
+    max_norm : ``float`` max norm of the gradients
```

Comment:
Can you indent the descriptions of the parameters on a new line, and use full sentences with capitals and full stops pleeeease